great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -87,7 +88,8 @@ def apply_dateutil_parse(column):
 
 
 class SparkDFExecutionEngine(ExecutionEngine):
-    """
+    """SparkDFExecutionEngine instantiates the ExecutionEngine API to support computations using Spark platform.
+
     This class holds an attribute `spark_df` which is a spark.sql.DataFrame.
 
```

Comment:
Can you add an `Args:` section here with the arguments that can be passed into the constructor (including any available via the parent)? 
Maybe you could also add a usage example with configuration? Similar to SlackNotificationAction for example: https://docs.greatexpectations.io/docs/reference/api/checkpoint/SlackNotificationAction